docs/pallas/grid_blockspec.md

```diff
@@ -33,7 +33,10 @@ for i in range(n):
 This generalizes to any tuple of integers (a length `d` grid will correspond
 to `d` nested loops).
 The kernel is executed as many times
-as `prod(grid)`. Each of these invocations is referred to as a "program".
+as `prod(grid)`.
+The default grid value `None` stands for `()`, and results in one
```

Comment:
Hmm can't we change the default to `()`? There's no reason for a user to ever pass in `None` and I would not mention that in the docs

Reply:
Indeed, I wanted to ask about that. I'd like to do that separately, because it changes the API.